models/DataObject/ClassDefinition/Data/Localizedfields.php

```diff
@@ -1089,7 +1089,7 @@ public function checkValidity($data, $omitMandatoryCheck = false)
             }
         }
 
-        if ($validationExceptions) {
+        if (count($validationExceptions)) {
```

Comment:
Shouldn't it not be bigger than 0?

Reply:
I improved the conditional to make it easier to understand

Reply:
`if (count($validationExceptions))` would have been alright, but anyway ;)